WcaOnRails/app/models/user.rb

```diff
@@ -238,6 +238,12 @@ def claim_wca_id_validations
     end
   end
 
+  # workaround / very nasty hotfix for https://github.com/rails/rails/issues/36965
```

Comment:
Just so it's easy to locate later :)

```suggestion
  # TODO: remove this workaround once https://github.com/rails/rails/issues/36965 is fixed
```